GitUI/Script/ScriptRunner.cs

```diff
@@ -142,7 +142,7 @@ private static CommandStatus RunScriptInternal(IWin32Window owner, IGitModule mo
 
             if (!scriptInfo.RunInBackground)
             {
-                bool success = FormProcess.ShowDialog(owner, argument, module.WorkingDir, null, true, process: command);
+                bool success = FormProcess.ShowDialog(owner, commands: null, argument, module.WorkingDir, null, true, process: command);
```

Comment:
Yes, we have a number of places where we ought to use an interface. This is a bigger work, and I expect to make some progress in the future.

Reply:
> Yes, we have a number of places where we ought to use interfaces instead of concrete types.
> This is a bigger work

Unfortunately, it is. `IGitUICommands.cs` lacks lots of accessed members of `GitUICommands`. Reverting for now.